src/main/lua/exasolrls/adapter_properties.lua

```diff
@@ -73,21 +76,19 @@ end
 --
 -- @raise validation error
 --
--- @return self for fluent programming
---
 function M:validate()
     if not self:has_value(M.SCHEMA_NAME_PROPERTY) then
-        error('F-RLS-PROP-1: Missing mandatory property "' .. M.SCHEMA_NAME_PROPERTY
-            .. '". Please define the name of the source schema.');
+        exaerror.create("F-RLS-PROP-1", "Missing mandatory property '" .. M.SCHEMA_NAME_PROPERTY .. "' ")
```

Comment:
Don't we have parameters as we did in Java?

Reply:
No, because the ones in Java come from the underlying Exasol VS. Mainly connection mode choices. There is only one possible mode here: local.